### DISCUSSION TITLE
Are there any plans enabling a movable Taskbar (left/right) in Win11 again after 24H2?

### DISCUSSION THREAD

**Discussion by p5rawq** [valinet/ExplorerPatcher#3321] [Q&A]
The main reason why I am using the ExplorerPatcher is to be able to move the Taskbar to the sides again. Which for me is crucial and it was working fine with 23H. But as we all know with Win11 24H2 they removed this hidden legacy function and therefore it is not available any more.
My question: are there any plans/ideas/workarounds to implement that feature again or is all hope lost and we have to wait until MS will or will not add this feature to Win11 directly?

If there is no hope I have to install Win10 on my new Laptop (with pre installed Win11), which at least would give me until Oct 2025 (and reconsider my option then).

**Comment by okphilo** [comment]
same opinion.

**Comment by pyrates999** [comment]
That would be up to Microsoft to implement it in the windows 11 taskbar.

**Comment by Gunhedo** [comment]
I only want to move my taskbar to another monitor, and not the main display.

**Comment by pyrates999** [comment]
Try this: https://www.youtube.com/watch?v=_3gnbZ_4QrE

**Reply by Gunhedo** [reply]
I know about this, but it would open most Apps/Games on that screen than, which I don't want.

**Reply by pyrates999** [reply]
EP can only enable features like this if Microsoft adds them first, and then for some reason, disables them.
